ui/raidboss/data/05-shb/eureka/delubrum_reginae_savage.js

```diff
@@ -317,6 +423,36 @@ export default {
       netRegexJa: NetRegexes.startsUsing({ source: 'スティギモロク・ロード', id: '57D8', capture: false }),
       response: Responses.getOut(),
     },
+    {
+      id: 'DelubrumSav Queen Empyrean Iniquity',
+      netRegex: NetRegexes.startsUsing({ source: 'The Queen', id: '59F9', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ source: 'Kriegsgöttin', id: '59F9', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ source: 'Garde-La-Reine', id: '59F9', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ source: 'セイブ・ザ・クイーン', id: '59F9', capture: false }),
+      condition: Conditions.caresAboutAOE(),
+      suppressSeconds: 1,
+      response: Responses.aoe(),
+    },
+    {
+      id: 'DelubrumSav Queen Cleansing Slash',
+      netRegex: NetRegexes.startsUsing({ source: 'The Queen', id: '59F5' }),
+      netRegexDe: NetRegexes.startsUsing({ source: 'Kriegsgöttin', id: '59F5' }),
+      netRegexFr: NetRegexes.startsUsing({ source: 'Garde-La-Reine', id: '59F5' }),
+      netRegexJa: NetRegexes.startsUsing({ source: 'セイブ・ザ・クイーン', id: '59F5' }),
+      condition: tankBusterOnParty,
+      // Probably this is where you swap, but maybe that's not something you can
+      // count on in an alliance raid, where there might not even be another tank.
+      response: Responses.tankBuster(),
```

Comment:
Maybe this needs "Double tankbuster on X" for tanks/healers?

Reply:
It's not possible to survive without immunity as there is a vulnerability up that occurs. However, In the logs I have where a tank gets hit and survives it because they "Dodge" the first hit (during immunity) and then get hit by the second.
It's odd, but all tanks with immunity will Dodge the first hit, and then if they are WAR or DRK, they get hit by the second which applies the Doom.

Reply:
I think most groups will be having tank invuln the ability and then swap after it ends. I have chosen to alter the comment about the trigger to clarify the spell behavior. Maybe a tank swap callout would be good so tanks swap after the second hit as WAR/DRKs would have a Physical Vulnerability Up and for the next Cleansing Slash a new tank is needed as the original tank's invulnerability is not up for it.

Reply:
Yeah, I'm not sure what the text here should be.  Is there a target for this attack? If so, and the player is a paladin / gnb maybe say "invuln tankbuster".  If not, maybe say "tankbuster + swap".  For other tanks or healers, I'm not sure what to say.